packages/adminrouter/docker/adminrouter-upstreams-ee.conf

```diff
@@ -1,17 +1,14 @@
 set $upstream_networking_api http://127.0.0.1:61430;
 
-proxy_ssl_trusted_certificate /run/dcos/pki/CA/certs/ca.crt;
```

Comment:
Yes, but not in this PR. There is an issue for that:https://jira.mesosphere.com/browse/DCOS-16017 We can/should probably also remove EE stuf from Open and vice-versa now that devkit containers have been split.

Let's properly pull it into a sprint and create a separate PR.